### ISSUE TITLE
Multiple stata kernels running together

### ISSUE THREAD

**Issue by shikharsingla** [kylebarron/stata_kernel#323]
**Is your feature request related to a problem? Please describe.**
If I run two do files together either from atom or one from atom and other from jupyter-notebook both stop running due to error since they both are accessing the same stata console on linux (I believe).

**Describe the solution you'd like**
Either stop the kernal from running the second do file if one is running already or create a new kernel. I am not sure if it is possible or not.

**Comment by kylebarron** [comment]
It should be possible to do this on Linux:
![image](https://user-images.githubusercontent.com/15164633/71426778-57d32400-266c-11ea-8da6-ed2712450e1c.png)

These two sessions are running on the same Linux machine but are independent.

**Comment by shikharsingla** [comment]
When I use stata_kernel from Jupyter or Atom, they seem to be running on same session and both codes stop running. Am i doing something wrong there?

**Comment by kylebarron** [comment]
Are you able to do the test I did above? Set a local in one and see if it works in the other session?

**Comment by kylebarron** [comment]
You also need to make sure you're creating a new session, not reusing the old one in Hydrogen. Look for a command like "Start new local session" in Hydrogen.

**Comment by shikharsingla** [comment]
okay thanks, I will give it a try!

**Comment by shikharsingla** [comment]
I tried and open stata console in two different terminal windows. They seem to be independent but somehow in atom or jupyter notebook code stops running due to some error if I open another jupyter-notebook and run some code there. I am a bit confused.

**Comment by kylebarron** [comment]
Is there an error message in the terminal you opened jupyter from?

**Comment by shikharsingla** [comment]
Yes, I so started two different jupyter-notebooks. I ran a code of around 150 lines in one and then ran the same code in the second one while the first one was still running. Second one produced 
destring n
variable n not found
for the same code. Same code in the first one ran without any errors. In fact, there is nothing called destring n in my do file. Everytime, it says some random variable is not found which is not even there in the do file. Thanks for your help.
